### PR TITLE
Quickswap: bugs. Feedback: not enough info about problems

### PR DESCRIPTION
**Pull request by NotEternal**
## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/swaponline/MultiCurrencyWallet/blob/master/docs/CONTRIBUTING.md) guide
- [ ] Good naming (as clear and simple as possible)
- [ ] Correct behavior if external API endpoints are down, return 404, 504 (or no answer), 401 errors (ddos simulation)
- [ ] I tested desktop/mobile resolution
- [x] I tested light/dark theme
- [ ] I tested different languages
- [ ] I checked the functionality once again (**AFFECT MONEY**)
- [ ] I checked the work on the Testnet
- [x] I checked the work on the Mainnet
- [ ] I checked the work in the plugin
- [ ] I checked the **PR** once again

## Tests

Please start auto tests as follows:

- add a label <ins>swap test</ins> to start swap tests
- add a label <ins>withdraw test</ins> to start withdraw tests

You can skip these tests if you completely sure that your changes aren't related to this functional

### Original issue
#4751

#4746
#4747

### CODE REVIEW

**Comment by developer24-exe** [comment]
builded https://testnet4762.swaponline.site ;

**Comment by developer24-exe** [comment]
commit 5dd301cd7 rebuilded https://testnet4762.swaponline.site https://mainnet4762.swaponline.site

File: src/front/shared/pages/Exchange/QuickSwap/index.tsx [hunk 1]
```diff
@@ -109,8 +109,14 @@ class QuickSwap extends PureComponent<IUniversalObj, ComponentState> {
   }
 
   componentDidMount() {
+    const { fromWallet } = this.state
+
     this.updateNetwork()
     actions.user.getBalances()
+
+    if (fromWallet.balance === 0) {
+      setTimeout(this.updateBalances, 1000)
```
Comment (noxonsu) [inline src/front/shared/pages/Exchange/QuickSwap/index.tsx:118]: а спама апи не будет?

**Comment by noxonsu** [comment]
- [ ] при переходе на exchange открывается атомик свап, а квик вообще задисеблен

**Comment by NotEternal** [comment]
> * [ ]  при переходе на exchange открывается атомик свап, а квик вообще задисеблен https://screenshots.wpmix.net/chrome_K3d1qNB72VKQQ5EHfcSEFq0LLpY1fWub.png

на тестнете не работает

File: src/front/shared/pages/Exchange/QuickSwap/index.tsx (hunk 1 above)
Comment (NotEternal) [reply src/front/shared/pages/Exchange/QuickSwap/index.tsx:118]: не думаю. Только один запрос при монтировании компонента

**Comment by developer24-exe** [comment]
commit 230dd90b2 rebuilded https://testnet4762.swaponline.site https://mainnet4762.swaponline.site

**Comment by developer24-exe** [comment]
commit a92855a3f rebuilded https://testnet4762.swaponline.site https://mainnet4762.swaponline.site

**Comment by developer24-exe** [comment]
commit 229607ad1 rebuilded https://testnet4762.swaponline.site https://mainnet4762.swaponline.site

File: src/front/client/index.html [hunk 2]
```diff
@@ -794,7 +794,7 @@ <h2 class="starter-modal__title">
       window.DEFAULT_FIAT = 'USD';
       window.DefaultCountryCode = '+1';
       window.widgetName = 'own widget name';
-      window.buyViaCreditCardLink = 'https://buy.moonpay.com/'; // need replace at 'https://buy.itez.com/swaponline' by default
+      window.buyViaCreditCardLink = 'https://buy.itez.com/'; // need replace at 'https://buy.itez.com/swaponline' by default
```
Comment (VitaliyShulik) [inline src/front/client/index.html:797]: remove comment

**Review by VitaliyShulik — APPROVED** [review]

**Comment by developer24-exe** [comment]
commit 99af7c76f rebuilded https://testnet4762.swaponline.site https://mainnet4762.swaponline.site

File: src/front/client/index.html (hunk 2 above)
Comment (NotEternal) [reply src/front/client/index.html:797]: for what? we still need to replace with **...itez.com/swaponline** =)

Comment (VitaliyShulik) [reply src/front/client/index.html:797]: itez can ban our link again)

**Pull request merged**